redistimeseries/src/test/java/io/github/dengliming/redismodule/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -225,6 +226,87 @@ public void testAggregationsAlign() {
         assertThat(end.get(1).getValue()).isEqualTo(10.0d);
     }
 
+    @Ignore("Only for redis timeseries >= 1.6.0")
```

Comment:
Can we open it for this PR? The timeseries version of ci is the lastest. So we can keep it open.

Reply:
latest release in redislabs is 1.14.3, you should use `redislabs/redistimeseries:1.6.0`